examples/go/example-datacenter.yaml

```diff
@@ -10,6 +10,31 @@ spec:
   cassandraImage: "gcr.io/cassandra-operator/cassandra:3.11.3"
   sidecarImage: "gcr.io/cassandra-operator/cassandra-sidecar:latest"
   imagePullPolicy: IfNotPresent
+  imagePullSecrets:
+    - name: regcred
+  backupSecretVolume:
+    secretName: backup-secret
+    #     type is a workaround for https://github.com/kubernetes/kubernetes/issues/68466
+    type: array
+    items:
+      - key: creds.json
+        path: creds.json
+  env:
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: "/etc/google/creds.json"
+  userSecretVolume:
+    secretName: dc1-user-secret
+  userConfigMapVolumeSource:
+    name: dc1-user-config
```

Comment:
Where is this config-map defined? The example should be complete.

Reply:
I basically copied the example from the original java code. I'll probably copy it back to the `doc` folder to add to one of the examples.

Reply:
added doc